### PR TITLE
kusama: General Admin to send Location mapping

### PR DESCRIPTION
**Pull request by muharem**
Fixes: https://github.com/polkadot-fellows/runtimes/issues/382

General Admin Origin to xcm `Location` mapping for outgoing messages

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

**Comment by muharem** [comment]
@bkchr @acatangiu FYI, I have added the tests and updated the changelog

File: relay/kusama/src/xcm_config.rs [hunk 1]
```diff
@@ -250,9 +252,15 @@ pub type StakingAdminToPlurality =
...
 
+/// Type to convert the `GeneralAdmin` origin to a Plurality `Location` value.
+pub type GeneralAdminToPlurality =
+	OriginToPluralityVoice<RuntimeOrigin, GeneralAdmin, GeneralAdminBodyId>;
+
 /// Type to convert a pallet `Origin` type value into a `Location` value which represents an
 /// interior location of this chain for a destination chain.
 pub type LocalPalletOrSignedOriginToLocation = (
+	// GeneralAdmin origin to be used in XCM as a corresponding Plurality `Location` value.
+	GeneralAdminToPlurality,
```
Comment (acatangiu) [inline relay/kusama/src/xcm_config.rs:263]: orthogonal to this PR, but quick Q: Kusama is also missing `TreasurerToPlurality`?

**Review by acatangiu — APPROVED** [review]

File: integration-tests/emulated/tests/people/people-kusama/src/tests/governance.rs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::*;
+use frame_support::sp_runtime::traits::Dispatchable;
+use kusama_runtime::governance::pallet_custom_origins::Origin::GeneralAdmin as GeneralAdminOrigin;
+
+#[test]
+fn general_admin_add_registrar() {
```
[the remark this answers is not in the document]
Comment (muharem) [reply integration-tests/emulated/tests/people/people-kusama/src/tests/governance.rs]: I left one test function `relay_commands_add_registrar` with a loop over two origins.

I personally do not like function abstraction when there is no clear (semantic) abstraction. This is often the case for tests. Frequently, we just move duplicated code into a function with a long name, which makes tests harder to read and maintain. I prefer copy-pasting over poor abstraction. Additionally, such abstractions introduce a variety of different styles to the codebase. Here I also prefer uniformity. A test framework usually helpful with it. In my case for example a data provider for a test function could be used.

File: relay/kusama/src/xcm_config.rs (hunk 1 above)
Comment (muharem) [reply relay/kusama/src/xcm_config.rs:263]: I do not see a use case for it now. In Polkadot we need it to manage Fellowship Treasury on Collectives.

**Review by bkchr — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Pull request merged**